diff --git a/dist/components/icon/icon.js b/dist/components/icon/icon.js
index 802175a456c882cd124fdc291d8b2ef8576a354c..7e1c250624a0fde03f54166e01916ea669fbcea1 100644
--- a/dist/components/icon/icon.js
+++ b/dist/components/icon/icon.js
@@ -77,8 +77,8 @@ const x = ({
       {
         alt: t,
         className: i(
+          !(e || l || s) && "bg-stone-800 dark:bg-stone-100",
           o,
-          !(e || l || s) && "bg-stone-800 dark:bg-stone-100"
         ),
         "data-component-part": "icon-image",
         src: m,
@@ -92,8 +92,8 @@ const x = ({
     "svg",
     {
       className: i(
+        !(e || l || s) && "bg-stone-800 dark:bg-stone-100",
         o,
-        !(e || l || s) && "bg-stone-800 dark:bg-stone-100"
       ),
       "data-component-part": "icon-svg",
       style: {
